cmd/minishift/cmd/addon/util.go

```diff
@@ -48,7 +48,7 @@ func GetAddOnManager() *manager.AddOnManager {
 func GetExecutionContext(ip string, routingSuffix string, ocPath string, kubeConfigPath string, sshCommander provision.SSHCommander) *command.ExecutionContext {
 	context, err := command.NewExecutionContext(ocPath, kubeConfigPath, sshCommander)
 	if err != nil {
-		glog.Errorln("Unable to initialise execution context.", err)
+		glog.Errorln("Unable to start the execution context.", err)
```

Comment:
Same as above. We are initialising the execution context, aka preparing it to be used. We are not starting anything.